content/advanced/330_servicemesh_using_appmesh/canary_deployment/deploy_canary.md

```diff
@@ -0,0 +1,51 @@
+---
+title: "Canary Deployment"
+date: 2020-01-27T08:30:11-07:00
+weight: 60
+draft: false
+---
+
+  
+Now lets deploy a new version (version 2) of Catalog Product Detail backend service. And change the `proddetail` Virtual Router to route traffic 50% to `proddetail-v1` version 1 and 50% to `proddetail-v2` version 2
```

Comment:
Typically, canaries roll out a bit more conservatively than 50/50. Maybe start with 5-10% traffic routed to v2, then gradually increase after observing application health? 

Also mention this is ideally automated, but ok to leave as manual observation at first.

Reply:
I wanted to show in the testing that in few clicks the next version will come as we use 50-50. If we change to 5-10%, they may need to keep clicking many times. I am ok with this. I can change it.